### DISCUSSION TITLE
How to list only ignored paths with ignore crate?

### DISCUSSION THREAD

**Discussion by Gumichocopengin8** [BurntSushi/ripgrep#2651] [Q&A]
Hello, 
I'm using `ignore` create in Rust.
How to just list ignored paths instead of non-ignored paths? I couldn't find an option to write the code.
This code does not work somehow. Am I missing something?

```rust
let mut gitignore_builder = GitignoreBuilder::new("./");
gitignore_builder.add("./.gitignore");
let gitignore = gitignore_builder.build().unwrap();
println!("gitignore, {:?}", gitignore.matched("./aaa/a.js", false).is_ignore());
```

Thank you.

**Comment by BurntSushi** [comment]
Sorry, but I don't understand the question. And I can't make heads or tails of your code because you haven't provided a reproducible example. I have no way to run your code in a way that gives me confidence that I'll see the same thing as you are. You also don't say what you see nor do you say what the expected result is.

Could you please flesh your question out with more detail?
